diff --git a/CMakeLists.txt b/CMakeLists.txt
index a379f54..b8c8188 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,7 +16,7 @@ enable_language(C)
 include(GNUInstallDirs)
 
 if(MSVC)
-set(OpenBLAS_LIBNAME libopenblas)
+set(OpenBLAS_LIBNAME openblas)
 else()
 set(OpenBLAS_LIBNAME openblas)
 endif()
@@ -186,9 +186,9 @@ if (NOT MSVC)
 endif()
 
 #build test and ctest
-add_subdirectory(test)
+#add_subdirectory(test)
 if(NOT NO_CBLAS)
-add_subdirectory(ctest)
+#add_subdirectory(ctest)
 endif()
 endif()
 
diff --git a/cmake/c_check.cmake b/cmake/c_check.cmake
index 56ae612..5eac269 100644
--- a/cmake/c_check.cmake
+++ b/cmake/c_check.cmake
@@ -59,7 +59,7 @@ endif ()
 # CMAKE_HOST_SYSTEM_PROCESSOR - The name of the CPU CMake is running on.
 #
 # TODO: CMAKE_SYSTEM_PROCESSOR doesn't seem to be correct - instead get it from the compiler a la c_check
-set(ARCH ${CMAKE_SYSTEM_PROCESSOR})
+set(ARCH ${CMAKE_HOST_SYSTEM_PROCESSOR})
 if (${ARCH} STREQUAL "AMD64")
   set(ARCH "x86_64")
 endif ()
